### PR TITLE
[APB-4149][PT]: Update Get All Invitations to Pass Query String of Services

### PR DESCRIPTION
**Pull request by JTY009**

### CODE REVIEW

File: app/uk/gov/hmrc/agentauthorisation/connectors/InvitationsConnector.scala [hunk 1]
```diff
@@ -67,12 +67,17 @@ class InvitationsConnector @Inject()(
   private[connectors] def cancelInvitationUrl(arn: Arn, invitationId: InvitationId) =
     new URL(baseUrl, s"/agent-client-authorisation/agencies/${arn.value}/invitations/sent/${invitationId.value}/cancel")
 
-  private[connectors] def getAgencyInvitationsUrl(arn: Arn, createdOnOrAfter: LocalDate): URL =
+  private[connectors] def getAgencyInvitationsUrl(
+    arn: Arn,
+    supportedServices: Seq[String],
+    createdOnOrAfter: LocalDate): URL = {
+    val services = s"service=${supportedServices.mkString(",")}&"
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentauthorisation/connectors/InvitationsConnector.scala]: this construction each time the above method is called , instead simply declare 
`val services = "HMRC-MTD-IT,HMRC-MTD-VAT`  and use in the URL straightaway

File: app/uk/gov/hmrc/agentauthorisation/models/StoredInvitation.scala [hunk 2]
```diff
@@ -39,6 +39,8 @@ case class StoredInvitation(
 
 object StoredInvitation {
 
+  val supportedStoredServices = Seq("HMRC-MTD-IT", "HMRC-MTD-VAT")
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentauthorisation/models/StoredInvitation.scala]: you can remove this and use val straightaway which is simple

File: app/uk/gov/hmrc/agentauthorisation/controllers/api/agent/AgentController.scala [hunk 3]
```diff
@@ -97,8 +97,7 @@ class AgentController @Inject()(
           invitationService
             .getInvitation(arn, invitationId)
             .map {
-              case pendingInv @ Some(PendingInvitation(pendingInvitation))
-                  if supportedServices.exists(pendingInvitation.service.contains) =>
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentauthorisation/controllers/api/agent/AgentController.scala]: do we stll need `supportedServices`?

Comment (JTY009) [reply app/uk/gov/hmrc/agentauthorisation/controllers/api/agent/AgentController.scala]: ah...

**Pull request merged**